src/components/Blocks/BlockDetail.jsx

```diff
@@ -3,44 +3,46 @@ import { useParams, Link } from 'react-router-dom'
 
 import { UserContext } from '../User/handleUser'
 
-import '../../common.css'
+// import '../../common.css'
 
 export default function BlockDetail () {
   const { height } = useParams()
   const [blockAttributes, setBlockAttributes] = useState(null)
   const { myAxios } = useContext(UserContext)
   useEffect(() => {
-    myAxios.get(process.env.REACT_APP_BACKEND_URL + `blocks/height/${height}`).then(res => {
+    myAxios.get(process.env.REACT_APP_BACKEND_URL + `blocks/height/${height}/`).then(res => {
       setBlockAttributes(prevInfo => res.data)
-    }).catch(err => console.log(err))
+    }).catch(err => console.log(err, 123))
```

Comment:
remove this logging or add meaningful errors.

Reply:
done